docs/cities/manchester/roles-and-duties.md

```diff
@@ -0,0 +1,111 @@
+---
+id: roles-and-duties
+title: Roles & Duties
+---
+
+Under the main trinity of [Voluneteer teams](volunteers/teams.md),namely Outreach, Education & Personal Development,
+generally, we need to act and fulfill different roles in order to run the course smoothly and effectively.
+These roles also help to keep each member being engadged and organised.
+
```

Comment:
I'd specify that these are mainly Education team roles/roles for the Saturday class - there are other tasks that need fulfilling outside of these on Outreach and PD. Want to avoid people feeling like Education volunteers are the 'main' volunteers and everyone else are 'other' volunteers if that makes sense!

Reply:
Good point! Have been wondering something similar, roles like lunch crew is not part any of them.

Reply:
Yeah I agree, some could be done by anyone, it may almost be worth having two lists. Happy to help you with organising it!

Reply:
Hi Lucy, the initiative text has been reworked, could you please give a review again.